### PR TITLE
ENH: Add black style pre-commit hook

### PR DESCRIPTION
**Pull request by thewtex**
Black is installed by the Superbuild and run on all changed Python
files during commit. A system version can also be used or configured
with the `black.executable` git configuration value.

### CODE REVIEW

File: src/Segmentation/Watersheds/SegmentWithWatershedImageFilter/Code.py [hunk 1]
```diff
@@ -42,6 +42,7 @@
           ' <InputFileName> <OutputFileName> <Threshold> <Level>')
     sys.exit(1)
 
+# aoeuaoe
```
Comment (dzenanz) [inline src/Segmentation/Watersheds/SegmentWithWatershedImageFilter/Code.py:48]: Mistake?

File: src/Segmentation/LevelSets/SegmentWithGeodesicActiveContourLevelSet/Code.py [hunk 2]
```diff
@@ -21,10 +21,13 @@
     print(
         "Usage: " + sys.argv[0] + "<InputFileName>  <OutputFileName>"
         " <seedX> <seedY> <InitialDistance> <Sigma> <SigmoidAlpha> "
-        "<SigmoidBeta> <PropagationScaling> <NumberOfIterations>")
+        "<SigmoidBeta> <PropagationScaling> <NumberOfIterations>"
```
Comment (dzenanz) [inline src/Segmentation/LevelSets/SegmentWithGeodesicActiveContourLevelSet/Code.py:24]: Is this file's formatting added to this commit as an example of what black does? Or is it unintentional?

**Review by dzenanz — APPROVED** [review]

**Review by jhlegarreta — APPROVED** [review]
Apart from Dženan's comment about the unintended comment on ` src/Segmentation/Watersheds/SegmentWithWatershedImageFilter/Code.py`, :+1:

**Review by mseng10 — APPROVED** [review]

File: src/Segmentation/Watersheds/SegmentWithWatershedImageFilter/Code.py (hunk 1 above)
Comment (thewtex) [reply src/Segmentation/Watersheds/SegmentWithWatershedImageFilter/Code.py:48]: Cruft from testing -- good catch :-)

File: src/Segmentation/LevelSets/SegmentWithGeodesicActiveContourLevelSet/Code.py (hunk 2 above)
Comment (thewtex) [reply src/Segmentation/LevelSets/SegmentWithGeodesicActiveContourLevelSet/Code.py:24]: Yes, this is from black.

**Pull request merged**